app/split_integer.py

```diff
@@ -2,6 +2,6 @@
     parts = []
     for parts_left in range(number_of_parts, 0, -1):
         next_number = value // parts_left
```

Comment:
Using integer division (`//`) might not distribute the value evenly across all parts. Consider revising the logic to account for any remainder to ensure even distribution.